CONTRIBUTING.md

```diff
@@ -87,50 +87,139 @@ You may need to install the dev_requirements for this
+
+`azext_iot_central_dns_suffix` is the DNS Suffix to use for the IoT Central Application. Optional variable, only used to populate --central-dns-suffix argument in IoT Central commands.
+
+`azext_iot_central_primarykey` is the IoT Central Application Id or name. Optional variable, used for IoT Central tests. Cannot be currently retrieved through the CLI.
+
+`azext_iot_central_storage_container` is the name of blob container belonging to the `azext_iot_teststorageaccount` storage account. Optional environment variable, defaults to 'central' when not specified.
+
+##### Digital Twin Test variables
+
+`azext_dt_region` is the region to restrict Digital Twins creation. Optional variable, if not specified, will create the Digital Twins in a supported region.
+
+`azext_dt_adx_cluster` is the name of the Azure Data Explorer Cluster to use. Optional variable, used for Digital Twin data history tests. Azure Data Explorer Cluster creation time can take up to 20 minutes, so having a cluster is recommended.
+
+`azext_dt_adx_database` is the name of the database in the Azure Data Explorer Cluster to use. Optional variable, used for Digital Twin data history tests.
+
+`azext_dt_adx_rg` is the resource group that contains the Azure Data Explorer Cluster. If not provided, `az_iot_testrg` will be used as the resource group.
+
+`azext_dt_ep_eventgrid_topic` is the Event Grid Topic to use. Optional variable, used for Digital Twin endpoint tests.
+
+`azext_dt_ep_servicebus_namespace` is the Service Bus Namespace to use. Optional variable, used for Digital Twin endpoint tests.
+
+`azext_dt_ep_servicebus_policy` is the policy for the topic in the Service Bus Namespace to use. Optional variable, used for Digital Twin endpoint tests.
+
+`azext_dt_ep_servicebus_topic` is the topic in the Service Bus Namespace to use. Optional variable, used for Digital Twin endpoint tests.
+
+`azext_dt_ep_eventhub_namespace` is the Event Hub Namespace to use. Optional variable, used for Digital Twin endpoint and data history tests.
+
+`azext_dt_ep_eventhub_policy` is the policy for the Event Hub instance in the Event Hub Namespace to use. Optional variable, used for Digital Twin endpoint and data history tests.
+
+`azext_dt_ep_eventhub_topic` is the Event Hub instance in the Event Hub Namespace to use. Optional variable, used for Digital Twin endpoint and data history tests.
+
+`azext_dt_ep_eventhub_topic_consumer_group` is the Service Bus Namespace to use. Optional variable, used for Digital Twin data history tests.
+
+`azext_dt_ep_rg` is the resource group that contains the endpoint (Event Hub, Event Grid, Service Bus) variables. Optional variable, if not provided, `az_iot_testrg` will be used as the resource group.
+
 ##### IoT Hub
 
 Execute the following command to run the IoT Hub integration tests:
 
-`pytest azext_iot/tests/iothub/ -k "_int"`
+`pytest azext_iot/tests/iothub/ -k "_int.py"`
```

Comment:
The `.py` suffix shouldn't be strictly necessary to run the IT's.

Reply:
without the ".py", some unit tests get caught because they have "_int" in the test name (although this only happens for central tests, I got used to adding the py):
![image](https://user-images.githubusercontent.com/73560279/161867657-38112a15-8b74-48a0-a22f-c6773162437f.png)

If we say that the keyword "_int" must only match unit tests, it will restrict naming unit tests so I prefer keeping the extension.

Reply:
Ok - the line I commented on has a path constraint for iothub, which doesn't have this issue (like you pointed out its currently an issue with central tests).

Then why include the extension path _only_ for a service-test area that doesn't have the issue? Further down there is still usage of `_int`

![image](https://user-images.githubusercontent.com/28658112/161871821-6c973789-d159-44a5-8edb-7d5fc4f58372.png)